packages/pluggableWidgets/datagrid-web/src/Datagrid.xml

```diff
@@ -43,6 +43,19 @@
                                 <caption>Content</caption>
                                 <description/>
                             </property>
+                            <property key="width" type="enumeration" defaultValue="autoFill">
+                                <caption>Column width</caption>
+                                <description />
+                                <enumerationValues>
+                                    <enumerationValue key="autoFill">Auto-fill</enumerationValue>
+                                    <enumerationValue key="autoFit">Auto-fit content</enumerationValue>
```

Comment:
Just `Fit content`?

Reply:
We are using same approach as Layout grid (Ctrl+C, Ctrl+V)